Sources/Examples/EchoNIO/Generated/echo.grpc.swift

```diff
@@ -39,15 +39,18 @@ internal final class Echo_EchoService_NIOClient: GRPCServiceClient, Echo_EchoSer
   internal let connection: GRPCClientConnection
   internal var serviceName: String { return "echo.Echo" }
   internal var defaultCallOptions: CallOptions
+  internal var errorDelegate: ClientErrorDelegate?
 
   /// Creates a client for the echo.Echo service.
   ///
   /// - Parameters:
   ///   - connection: `GRPCClientConnection` to the service host.
   ///   - defaultCallOptions: Options to use for each service call if the user doesn't provide them.
-  internal init(connection: GRPCClientConnection, defaultCallOptions: CallOptions = CallOptions()) {
+  ///   - errorDelegate: Delegate to use when receiving errors.
+  internal init(connection: GRPCClientConnection, defaultCallOptions: CallOptions = CallOptions(), errorDelegate: ClientErrorDelegate? = nil) {
```

Comment:
Should we use a "logging client error delegate" by default, similar to the server case?

Reply:
I wasn't sure it made as much sense on the client as it does on the server but I don't feel strongly about it either way.

Reply:
Makes sense. How about an error delegate that logs in debug builds (to ease development and help us help users encountering issues), but not release builds? I think the NIO team has a function we could steal that does something like that.

Reply:
Sounds good!